### PR TITLE
Add GetAccessibleCustomers example

### PR DESCRIPTION
**Pull request by AnashOommen**

### CODE REVIEW

File: examples/Services/AccountManagement/GetAccessibleCustomers.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+using System;
+
+namespace Google.Ads.GoogleAds.Examples.V0
+{
+    /// <summary>
+    /// This code example lists the resource names for the customers that the authenticating user
+    /// has access to. This example may can be used to display a list of customer IDs that can be
+    /// selected by the user for setting the login-customer-id in a subsequent call. See
+    /// https://developers.google.com/google-ads/api/docs/concepts/call-structure#login-customer-id
+    /// to learn more about login-customer-id header.
```
Comment (jradcliff) [inline examples/Services/AccountManagement/GetAccessibleCustomers.cs]: about -> about the

File: examples/Services/AccountManagement/GetAccessibleCustomers.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        public override string Description
+        {
+            get
+            {
+                return "This code example lists the resource names for the customers that the " +
+                    "authenticating user has access to. This example may be used to display a " +
+                    "list of customer IDs that can be selected by the user for setting the " +
+                    "login-customer-id in a subsequent call. See "+
+                    "https://developers.google.com/google-ads/api/docs/concepts/call-structure#login-customer-id "+
+                    "to learn more about login-customer-id header.";
```
Comment (jradcliff) [inline examples/Services/AccountManagement/GetAccessibleCustomers.cs]: about -> about the

File: examples/Services/AccountManagement/GetAccessibleCustomers.cs [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                // Retrieve the list of customer resources.
+                string[] customerResourceNames = customerService.ListAccessibleCustomers();
+
+                Console.WriteLine($"Total results: {customerResourceNames.Length}.");
+
+                // Display the result.
+                int i = 1;
+                foreach (string customerResourceName in customerResourceNames)
+                {
+                    Console.WriteLine($"{i++}) Customer resource name: {customerResourceName}");
```
Comment (jradcliff) [inline examples/Services/AccountManagement/GetAccessibleCustomers.cs]: Let's be consistent about how we print lists in examples (I mentioned this on the other PR as well).

Comment (AnashOommen) [reply examples/Services/AccountManagement/GetAccessibleCustomers.cs]: Done.

File: examples/Services/AccountManagement/GetAccessibleCustomers.cs (hunk 1 above)
Comment (AnashOommen) [reply examples/Services/AccountManagement/GetAccessibleCustomers.cs]: Done

File: examples/Services/AccountManagement/GetAccessibleCustomers.cs (hunk 2 above)
Comment (AnashOommen) [reply examples/Services/AccountManagement/GetAccessibleCustomers.cs]: Done

File: examples/Services/AccountManagement/GetAccessibleCustomers.cs (hunk 3 above)
Comment (AnashOommen) [reply examples/Services/AccountManagement/GetAccessibleCustomers.cs]: Done

**Pull request merged**